nifi-nar-bundles/nifi-salesforce-bundle/nifi-salesforce-processors/src/main/java/org/apache/nifi/processors/salesforce/QuerySalesforceObject.java

```diff
@@ -194,9 +193,8 @@ public class QuerySalesforceObject extends AbstractProcessor {
     static final PropertyDescriptor INITIAL_AGE_FILTER = new PropertyDescriptor.Builder()
             .name("initial-age-filter")
             .displayName("Initial Age Filter")
```

Comment:
Based on the description, it may be better to change the property display name to something like "Initial Age Start Time", since it does not apply to the initial end time.
```suggestion
            .displayName("Initial Age Start Time")
```